BuildLoader.py

```diff
@@ -122,14 +126,14 @@ def __init__(self, *args, **kwargs):
 
         # NOTE: Variables starting with '_' will not be exported to Platform.dsc
 
-        # Define default private key (PEM format) used to sign configuration data, firmware capsule and IAS image
-        key_dir = os.path.join('BootloaderCorePkg', 'Tools', 'Keys')
+        # Default key dir is set by SLIMBOOT_KEY_DIR. _KEY_DIR is set to NULL.
+        self._KEY_DIR = ''
         # Allow master key to be able to sign everything by default
         self._MASTER_KEY_USAGE      = HASH_USAGE['PUBKEY_CFG_DATA'] | HASH_USAGE['PUBKEY_FWU'] | HASH_USAGE['PUBKEY_OS'] | \

```

Comment:
@sagraw2:
Since the master key can also be used to sign CFG_DATA/FWU/OS by default,   should we remove it to be consistent with   your documentation ?

Reply:
Yes. That would align our 'default' implementation with the documentation.

Reply:
Removed self._MASTER_KEY_USAGE. 
Pub key hashes would be defined in boardconfig.py.